geometry_msgs/Point currentLocation
geometry_msgs/Point[] plannedRoute
float32[] plannedKappa
std_msgs/Float32 plannedVelocity